docs/how-to/namespace-transition.md

```diff
@@ -94,6 +146,36 @@ release from the `knative-gcp` repository. We will update the page with specific
    kubectl apply --filename https://github.com/google/knative-gcp/releases/download/${KGCP_VERSION}/cloud-run-events.yaml
    ```
 
+1. Update Workload Identity authentication to work in the new namespace. This
+   can only be done after the upgrade above is executed, otherwise the
+   Kubernetes Service Accounts in the new namespace will not be automatically
+   created.
+
+   ```shell script
+   # auth for the control plane
+   gcloud iam service-accounts add-iam-policy-binding \
+     --role roles/iam.workloadIdentityUser \
+     --member serviceAccount:$PROJECT_ID.svc.id.goog[$NEW_EVENTS_NS/controller] \
+     $CONTROLLER_GSA@$PROJECT_ID.iam.gserviceaccount.com
+   kubectl annotate serviceaccount -n $NEW_EVENTS_NS controller \
+     iam.gke.io/gcp-service-account=$CONTROLLER_GSA@$PROJECT_ID.iam.gserviceaccount.com
+
+   # auth for the broker data plane
+   gcloud iam service-accounts add-iam-policy-binding \
+     --role roles/iam.workloadIdentityUser \
+     --member=serviceAccount:$PROJECT_ID.svc.id.goog[$NEW_EVENTS_NS/broker] \
+     $BROKER_GSA@$PROJECT_ID.iam.gserviceaccount.com
+   kubectl annotate serviceaccount -n $NEW_EVENTS_NS broker \
+     iam.gke.io/gcp-service-account=$BROKER_GSA@$PROJECT_ID.iam.gserviceaccount.com
+
+   # mark config-gcp-auth as initialized (important for the Cloud Console, which checks for this)
+   kubectl annotate configmap  -n $NEW_EVENTS_NS  \
+     config-gcp-auth --overwrite events.cloud.google.com/initialized=true
+
+   ```
+
+### Verify and Correct Existing Functionality
+
 1. Confirm the new BrokerCell is working correctly.
 
    First ensure that the BrokerCell exists inside the `events-system` namespace.
```

Comment:
Comment for line not edited in this PR:

```
1. Delete the old BrokerCell since it is no longer useful.
```

What happens if you just delete the BrokerCell itself? `kubectl delete brokercell default -n cloud-run-events` K8s GC may delete all the deployments, services, and HPAs for you.

Reply:
Updated to a single command.